scripts/publish-docker.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+set -x
+
+if [ ! "$#" -ne 4 ] ; then
+    echo "usage: $0 LOG_NAME BASE_TAG HASH VERSION" >&2
+    exit 1
+fi
+
+log_name="$1"
+base_tag="$2"
+hash="$3"
+version="$4"
+
+underscore_name="$(echo -n "$log_name" | tf - _)"
```

Comment:
blocking: `tr`, I think you mean (or `underscore_name="${log_name//-/_}"` if this were Bash).

Reply:
lmao the muscle memory